app/tasks/maintenance.php

```diff
@@ -118,24 +118,33 @@ function renewCertificates($dbForConsole)
             }
         }
 
+        function filesCacheKeyEviction()
+        {
+            /**
+              *  Delete cache keys who did not modify for the last 90 days.
+             */
+            shell_exec('find '.APP_STORAGE_CACHE.' -type f ! -mtime -90 -print0|xargs -0 rm -f');
```

Comment:
We probably need to check the performance on like 1 million files 🤔

Reply:
The current way would hit the max length limit for commands, causing the truncated filenames leading to weird not-found errors.

If there was no special reason to use `rm -f`, like files created with wrong permissions or user, using find's `-delete` switch should work _pretty much_ regardless of the amount of files.

Long runtimes must be considered though.